backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -364,16 +346,15 @@ extern "C" {
     /// Perform the negation of a u64 input LWE ciphertext vector.
     /// - `v_stream` is a void pointer to the Cuda stream to be used in the kernel launch
     /// - `gpu_index` is the index of the GPU to be used in the kernel launch
-    /// - `lwe_array_out` is an array of size
-    /// `(input_lwe_dimension + 1) * input_lwe_ciphertext_count` that should have been allocated on
-    /// the GPU before calling this function, and that will hold the result of the computation.
-    /// - `lwe_array_in` is the LWE ciphertext vector used as input, it should have been
-    /// allocated and initialized before calling this function. It has the same size as the output
-    /// array.
+    /// - `lwe_array_out` is an array of size `(input_lwe_dimension + 1) *
+    ///   input_lwe_ciphertext_count` that should have been allocated on the GPU before calling this
+    ///   function, and that will hold the result of the computation.
+    /// - `lwe_array_in` is the LWE ciphertext vector used as input, it should have been allocated
+    ///   and initialized before calling this function. It has the same size as the output array.
```

Comment:
@mayeul-zama does rust analyzer automatically or does rustfmt automatically format this ?

Reply:
IIRC I tried `cargo clippy --fix` and it didn't work

Reply:
rust fmt by any chance ? and so does it auto indent with rust analyze ?

otherwise I'm inclined to ignore that lint as it will just be a nuisance

Reply:
No

I think it's more readable now
I don't think it will cause significant friction, only 19 files are affected

Reply:
does not change anything in docs.rs and if we have to do it manually it's going to be super annoying

Reply:
how did you fix those then, manually ?

Reply:
Yes

The time to format new doc should be much smaller than the one to write it

Reply:
I'll see when using it, but wasting time on that is really unnecessary as it gets processed into an HTML anyways, hopefully they include some auto fix for clippy of cargo fmt